README.md

```diff
@@ -53,9 +53,9 @@ We care a lot about speed and that is why we built it on [Vert.x](http://vertx.i
 In order to separate static content and dynamic data we introduced a Templating Engine, which merges a template obtained from the content repository and dynamic data provided by microservices using [Handlebars.js](http://handlebarsjs.com/). Here is what a template looks like:
 
 ```html
-<script data-api-type="templating" data-call-uri="/path/to/service.json" type="text/x-handlebars-template">
-    <h2>{{header}}</h2>
-    <div>{{body.content}}</div>
+<script data-api-type="templating" data-uri-dataservice="/path/to/service.json" type="text/x-handlebars-template">
```

Comment:
It is possible to omit method. By default _-all-_ is applied. If doc does not say about this feature, it should be updated.